src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -216,11 +216,13 @@ function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, pol
         }
     }
 
-    // Update the data that is returned from back-end to draft value
-    const draftStep = reimbursementAccount.draftStep;
-    if (draftStep) {
-        BankAccounts.updateReimbursementAccountDraft(getBankAccountFields(getFieldsForStep(draftStep)));
-    }
+    useEffect(() => {
+        // Update the data that is returned from back-end to draft value
+        const draftStep = reimbursementAccount.draftStep;
+        if (draftStep) {
+            BankAccounts.updateReimbursementAccountDraft(getBankAccountFields(getFieldsForStep(draftStep)));
+        }
+    }, [getBankAccountFields, reimbursementAccount.draftStep]);
```

Comment:
unfortunately, it doesn't work. We need more conditions. Looks like it's a mistake after migration from the class component - this condition should be in another useEffect after few more checks: https://github.com/Expensify/App/pull/29166#discussion_r1372472415

This is a PR when this part was originally added
https://github.com/Expensify/App/pull/28305/files

Reply:
and please fix lint ;)